release-note.md

```diff
@@ -0,0 +1,44 @@
+# 2.1.1 Release
```

Comment:
Hi, thanks for your working on this patch, you don't need to submit this file, this file is only used in [2.1.1 release branch](https://github.com/apache/incubator-seatunnel/tree/2.1.1-prepare)